CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3742,7 +3770,97 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd \"${BUILT_PRODUCTS_DIR}\"\n\nlipo \\\n\t\"${BUILD_DIR}/${CONFIGURATION}-appletvos/libCrashReporter-AppleTVOS.a\" \\\n\t\"${BUILD_DIR}/${CONFIGURATION}-appletvsimulator/libCrashReporter-AppleTVSimulator.a\" \\\n  -create -output ${EXECUTABLE_PATH}\n\nln -sfh Versions/${CURRENT_VERSION}/${PRODUCT_NAME} ${FULL_PRODUCT_NAME}/${PRODUCT_NAME}\n";
+			shellScript = "lipo \\\n\t\"${BUILD_DIR}/${CONFIGURATION}-appletvos/lib${PRODUCT_NAME}-appletvos.a\" \\\n\t\"${BUILD_DIR}/${CONFIGURATION}-appletvsimulator/lib${PRODUCT_NAME}-appletvsimulator.a\" \\\n  -create -output \"${BUILT_PRODUCTS_DIR}/lib${PRODUCT_NAME}-tvOS.a\"\n";
```

Comment:
http://mergely.com/cIV37lZo/

Reply:
Why does it require to have a lot of new scripts? Can you give me some contexts?

Reply:
this one is in framework target (one line `lipo`) to create framework from 2 static libs.
The actions are very similar that we had in AC but it's organized/splitted a bit different. Actually I like many 1 liners more than one long script in AC.